usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/PetData.java

```diff
@@ -110,6 +135,106 @@ public void setRecommendedKcal(Double recommendedKcal) {
         this.recommendedKcal = recommendedKcal;
     }
 
+    /**
+     * Checks that field has the correct format for a Pet simple attribute.
+     * @param field Name of the attribute.
+     */
+    public static void checkSimpleField(String field) {
+        if (!GENDER.equals(field) && !BREED.equals(field) && !BIRTH.equals(field)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Boolean expression complexity is 5 (max allowed is 3).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5479466)